portfolio/src/main/webapp/style.css

```diff
@@ -1,65 +1,78 @@
 .background {
   background-color: #ff9999;
 }
-
+ 
 hr.bar {
   border: 1px solid black;
 }
-
+ 
 .title {
   text-align: center;
+  font-family: "Georgia";
```

Comment:
Noticed that you are setting the font family to Georgia for multiple elements in this file. 

There's a cleaner way to set the font (and other attributes) for the entire page:
Since HTML is in a tree-like structure, when you set an attribute in a parent, the children should inherit that style as well as long as they don't manually override it. In the case of the font, if you add a css for .body{} and set the font there, it should set it for the entire page since the html body tag wraps all of the elements you have defined the font for here.

Please do test this out first as it's been a while since I've touched CSS. But I do think this should work :)

Also nit: use single (') over double (") quotations in CSS
https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks 
Please do the same for entire file

Reply:
It worked! I will keep that in mind for the future as well. Thanks Linh :)